sunpy/net/attr.py

```diff
@@ -92,24 +94,37 @@ def __dir__(self):
         """
         return super().__dir__() + self._attr_registry[self].name
 
+    def __repr__(self):
+        """
+        Returns the normal repr plus the pretty attr __str__.
+        """
+        return object.__repr__(self) + "\n" + str(self)
+
     def __str__(self):
```

Comment:
We might want to consider moving the print code out into its own function so the code in clients and Fido could reuse it.

Reply:
So you want a private function that will just print out all attrs that can be used anywhere?